disputer/disputer.js

```diff
@@ -63,8 +64,9 @@ class Disputer {
     await this.priceFeed.update();
   }
 
-  // Queries disputable liquidations and disputes any that were incorrectly liquidated.
-  async queryAndDispute() {
+  // Queries disputable liquidations and disputes any that were incorrectly liquidated. If `disputerOverridePrice` is
+  // provided then the disputer will ignore the price feed and use the override price instead for all undisputed liquidations.
+  async queryAndDispute(disputerOverridePrice = null) {
```

Comment:
Do you have to explicitly set the default `=null` here? I think if you don't provide a default value, and its not passed in, it will be null?

Reply:
It will technically be `undefined`, but that will still work for the ternary below.

Reply:
you right @nicholaspai it's not really required. I just wanted to make it clear that in almost all cases it's null. In the testing, for example, we simply don't provide this value to this function.